controller/gke-cluster-config-handler.go

```diff
@@ -649,6 +649,7 @@ func BuildUpstreamClusterState(cluster *gkeapi.Cluster) (*gkev1.GKEClusterConfig
 				LocalSsdCount: np.Config.LocalSsdCount,
 				MachineType:   np.Config.MachineType,
 				Preemptible:   np.Config.Preemptible,
+				Tags:          np.Config.Tags,
```

Comment:
Does the provider add any tags automatically after the cluster is created?

Reply:
No, it does not. But this is still needed if the user added their own tags at creation.